### ISSUE TITLE
Minor issue: It took three tries to initially start a game on playkey.net

### ISSUE THREAD

**Issue by terric** [vilkoz/playkey-linux-launcher#2]
**Minor issue: You initially have to start a game three times, before it works**

Since the readme doesn't say anything on how to actually use this and involves some guesswork, I thought I'd post a little more info here.

After successfully installing the package, I could simply type `playkey-linux`
into a terminal which opens a chromium Window to PlayKey's website.  

After setting up your account and paying for streaming time, you can log in, search for a game you want to play and click the button to start it.  

It took three tries to actually start a game.  
The game I was trying to play was Fortnite, which is very popular and not available on Linux, so it seemed like a good one to test the Service.

1. It doesn't work on the first try.  
For me it first tried to re-download the installer and I thought it didn't work.
I deleted the installer from my downloads, since it was redundant. 

2. Then I pressed the "start game" button again and got a bit further. It opened a new Window and showed a loading screen, then it crashed.

3. The third time I typed `playkey-linux` into the terminal again, signed in, pressed the button, and I got all the way to the VirtualPC login for my game account.

**Then I was able to play the game**
I got to login to my EpicGames account which I already had beforehand.  
The game started up and miraculously I was actually able to play Fortnite on my low-end Linux laptop with good performance and visuals.  
Latency is there, but surprisingly low.  
Visuals are a bit compressed, but still look decent.
3$ for 3 Hours of gaming. Worth it in my opinion, if you don't have access to a beefy Windows PC otherwise.

From then on, to start the game it only took one click of the "start game" button for things to actually work. 

So to summarize the issue: It seems that the PlayKey site doesn't initially detect that you have the PlayKey client-Software installed. It takes a couple tries to start a game.

But in the end it all works out. Thanks a lot for this convenient package!  
Feel free to immediately close this issue. I just wanted to post some extra info on how to use this.

**Comment by vilkoz** [comment]
The Playkey site actually has no way to check if you have some program installed on your computer, so they just show you overlay if you enter to their website from clean browser, they detect it by setting cookie `isHasClient`, as you can see in the following code snippet from playkey.net:
```js
// webpack://./src/app/games/components/download/donwload.component.ts

  // метод запускает игру или делает роут на страницу скачивания если у клиенты нет куки
  public playOrShowDownloadClick(params: IPlayConfig): void {
    this.getResponseParams(params).subscribe(() => {
      if (this._cookie.get('isHasClient') !== undefined) {
        this.runGame();
      } else {
        //  ройт на страницу скачивания клиента игры
        this.downloadClientRouter();
        if (this.isRemoteClickDemo) {
          this._analytic.event('user', 'btb onboarding opened download app g_witcher', 'Etalon.send', `timestamp:${Date.now()}`);
        } else {
          this._analytic.event('game', `click_play_and_show_download;${params.code}`);
        }
      }
    });
  }
```
Fixed this with the modification to the chrome-extension.
